docker/Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN set -x \
     && rm -rf "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz.sha512" \
     # install zookeeper shell
     && wget -q https://bootstrap.pypa.io/2.7/get-pip.py \
+    && python --version \
```

Comment:
If python3 is installed, you should update pip 3 too at line 50.

Reply:
This line returns "2.7"

I tried to move to pip3, but I need pip 2 because we need to install zookeeper-shell and with pip3 it fails

Reply:
I wonder why it fails. I was able to run `pip3 install zk-shell` successfully from inside the latest `python` docker image. Given that python 2.7 is deprecated, I think it might be worth creating an issue to upgrade the version of python shipped in the docker image.

Reply:
LGTM